_source/logzio_collections/_log-sources/azure-blob.md

```diff
@@ -139,14 +139,14 @@ where you'll configure the resources to be deployed.
 |---|---|---|
 | Resource group | Select your existing resource group. | Required |
 | Location | Select the same region as the Azure services that will stream data to this Blob Storage.  |  Required |
-| Logzio host | {% include log-shipping/listener-var.md %} |  Required |
```

Comment:
Best practice is to retain this snippet, rather than manually replacing the single sourced text. 
Would to relevant to understand how the Blob Storage Account Name text got lost.